app/code/Kogut/ProductsGeneration/Model/GenerationItem.php

```diff
@@ -0,0 +1,19 @@
+<?php
+
+declare(strict_types=1);
+
+namespace Kogut\ProductsGeneration\Model;
+
+use Magento\Framework\Model\AbstractModel;
+
+class GenerationItem extends AbstractModel
```

Comment:
Create getters and setters in class annotations
Like
https://github.com/magento/magento2/blob/2.4-develop/app/code/Magento/User/Model/User.php#L25

Reply:
Fixed